test/ValidateID.cpp

```diff
@@ -823,6 +823,40 @@ TEST_F(ValidateID, OpLoadPointerBad) {
 )";
   CHECK(spirv, SPV_ERROR_INVALID_ID);
 }
+TEST_F(ValidateID, OpLoadBitcastPointerGood) {
+  const char* spirv = R"(
+%1  = OpTypeVoid
+%2  = OpTypeInt 32 1
+%9  = OpTypeFloat 32
+%3  = OpTypePointer UniformConstant %2
+%10 = OpTypePointer UniformConstant %9
+%4  = OpTypeFunction %1
+%6  = OpFunction %1 None %4
+%7  = OpLabel
+%11 = OpBitcast %10 %3
+%8  = OpLoad %9 %11
+      OpReturn
+      OpFunctionEnd
+)";
+  CHECK(spirv, SPV_SUCCESS);
+}
+TEST_F(ValidateID, OpLoadBitcastNonPointerBad) {
+  const char* spirv = R"(
+%1  = OpTypeVoid
+%2  = OpTypeInt 32 1
+%9  = OpTypeFloat 32
+%3  = OpTypePointer UniformConstant %2
+%4  = OpTypeFunction %1
+%6  = OpFunction %1 None %4
+%7  = OpLabel
+%8  = OpLoad %2 %3
```

Comment:
Oh wait, this only concerns Logical addressing, not Physical*. Even then, isn't OpFunctionParameter a pointer as well? And OpBitcast isn't restricted to Phyiscal, so it can "create" pointers as well. Really depends on how you define "create".